docs/DEVELOPER_GUIDE.md

```diff
@@ -57,7 +57,7 @@ KFServing currently requires `Knative Serving` for auto-scaling, canary rollout,
 * You can follow the instructions on [Set up a kubernetes cluster and install Knative Serving](https://knative.dev/docs/install/) or
 [Custom Install](https://knative.dev/docs/install/knative-custom-install) to install `Istio` and `Knative Serving`. Observability plug-ins are good to have for monitoring.
 
-* If you already have `Istio` (e.g. from a Kubeflow install) then simply skip the `Istio` steps. For Kubeflow install, you can install `Knative Serving` v0.8 via
+* If you already have `Istio` or `Knative` (e.g. from a Kubeflow install) then you don't need to install them explictily. From Kubeflow v0.7 onwards, KNative 0.8 and Istio 1.1.17 are installed by default as part of the Kubeflow installation. For prior versions of Kubeflow, you can install `Knative Serving` v0.8 via
```

Comment:
`kubeflow/manifests` may soon be changed to v0.10 though.

Reply:
@yuzisun take a look - simplified the verbiage more, and removed any reference to manifests, as it was causing more confusion

Reply:
I think the istio version in 0.7 is 1.1.6 according to here https://github.com/kubeflow/manifests/blob/v0.7-branch/istio/istio-install/base/kustomization.yaml ?

Reply:
@yuzisun corrected and rebased

Reply:
/1.1.16/1.1.6/

Reply:
done